megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -1503,9 +1505,28 @@ private static String toHitIsImpossible(IGame game, Entity ae, int attackerId, T
                 return Messages.getString("WeaponAttackAction.TooLowForFrontSide");
             }
             // Aft-mounted weapons can only be fired at targets at least 1 altitude lower
-            if ((weapon.getLocation() == Aero.LOC_AFT)) {
-                if (ae.isAirborne() && (altDif > -1)) {
+            // For grounded spheroids, weapons can only be fired at targets in occupied hexes, 
+            // but it's not actually possible for a unit to occupy the same hex as a grounded spheroid so
+            // we simplify the calculation a bit
+            if (weapon.getLocation() == Aero.LOC_AFT) {
+                if(altDif > -1) {
                     return Messages.getString("WeaponAttackAction.TooHighForAft");
+                } 
+                
+                // if both targets are on the ground
+                // and the target is below the attacker 
+                // and the attacker is in one of the target's occupied hexes
+                // then we can shoot aft weapons at it
+                // note that this cannot actually happen in MegaMek currently but is left here for the possible eventuality
+                // that overhanging dropships are implemented
+                if(!ae.isAirborne() && !target.isAirborne()) {
+                    boolean targetInAttackerHex = ae.getOccupiedCoords().contains(target.getPosition()) ||
+                            ae.getPosition().equals(target.getPosition());
+                    boolean targetBelowAttacker = ae.getElevation() > target.getElevation();
```

Comment:
getElevation() returns the elevation relative to the surface of the hex rather than the absolute level.
```suggestion
                    boolean targetBelowAttacker = game.getBoard().getHex(ae.getPosition()).surface() >
                        game.getBoard().getHex(ae.getPosition()).surface() + target.getElevation();

```

Reply:
Seems like, mathematically, this could be reduced to 0 > target.getElevation() (i.e. the target is below the surface of the hex), which I don't think is what we're looking for. Unless I'm missing something?

Reply:
That was my first thought, but then I realized that the dropship elevation is always zero over the center hex, but the situation implies that the other unit is in one of the surrounding hexes which has a lower surface elevation. This is why you need to compare `surface()` but also include the target rlevation because it could be a vtol flying above the dropship.

Reply:
Ah, I think I get it. Probably the second surface call would be for the target's position then.

Reply:
Oh, I typed that wrong. Sorry for the confusion.

Reply:
Done